constants.py

```diff
@@ -1,2 +1,2 @@
-windowWidth = 750
-windowHeight = 650
```

Comment:
The window has become even smaller, but since this is just temporary, I guess it's fine for now.
We need to settle on one fixed larger arena size in the future.